PokedexMentoria/Components/Pokeball.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Pokeball.swift
+//  PokedexMentoria
+//
+//  Created by Matheus Rodrigues Araujo on 06/01/23.
+//
+
+import UIKit
+
+class PokeBallView: UIView {
+    
+    // MARK: Variables
+    var viewCenter: CGPoint {
```

Comment:
sempre que as variáveis e métodos só tiverem visibilidade pra própria classe, acho que dava pra fazer private

Reply:
Feito.
Inclusive, uma pergunta, nessa logica, toda variavel calculada vai ser sempre private?

Reply:
não necessariamente toda, mas toda que for ser acessada só internamente, acho que todo método e variável merece as perguntas "Outra classe precisa ter acesso a isso?",  "Eu quero que o usuário consiga alterar isso?" (nessa segunda fica bem evidente em Design System, pq vc não quer por exemplo que o usuário mude a cor do botão dentro de um card por exemplo)